Backend/router/v1/report.ctrl.js

```diff
@@ -10,32 +10,42 @@ const createReport = (req, res) => {
   }).then((result) => {
     res.status(201);
     res.send("good");
+  }).catch((err) => {
+    res.json({msg: "error"});
```

Comment:
what error? will the frontend be able to know the type of error just be seeing 'error'? Please specify.